closed/openj9_version_info.h.in

```diff
@@ -30,5 +30,6 @@
 #define J9VERSION_STRING          "@JRE_RELEASE_VERSION@"
 #define OPENJDK_SHA               "@OPENJDK_SHA@"
 #define OPENJDK_TAG               "@OPENJDK_TAG@"
+#define J9JVM_VERSION_STRING      "@OPENJ9_VERSION_STRING@"
 
 #endif /* OPENJ9_VERSION_INFO_H */
```

Comment:
C source files should have a final newline - especially header files.

Reply:
Fixed.

Reply:
@keithc-ca pls review again.